packages/plugin-ext/src/common/types.ts

```diff
@@ -24,3 +24,13 @@ export function isObject(obj: any): boolean {
         && !(obj instanceof RegExp)
         && !(obj instanceof Date);
 }
+
+export enum LogType {
```

Comment:
what if there are debug traces ?

Reply:
There is could be even more stuff. I think we should add them on demand if we don't know whole list for now.